include/substrait-mlir/Dialect/Substrait/IR/SubstraitAttrs.td

```diff
@@ -67,6 +67,28 @@ def Substrait_DateAttr
   let assemblyFormat = [{ `<` $value `>` }];
 }
 
+def Substrait_IntervalDaySecondAttr
+    : Substrait_StaticallyTypedAttr<"IntervalDaySecond", "interval_day_second",
+                                    "IntervalDaySecondType"> {
+  let summary = "Substrait interval day to second type";
+  let description = [{
+    This type represents a substrait interval day to second attribute type.
+  }];  
+  let parameters = (ins "int32_t":$days_value, "int32_t":$seconds_value);
+  let assemblyFormat = [{ `<` $days_value `` `d` $seconds_value `` `s` `>` }];
+}
+
+def Substrait_IntervalYearMonthAttr 
+    : Substrait_StaticallyTypedAttr<"IntervalYearMonth", "interval_year_month",
+                                    "IntervalYearMonthType"> {
+  let summary = "Substrait interval year to month type";
+  let description = [{
+    This type represents a substrait interval year to month attribute type.
+  }];  
+  let parameters = (ins "int32_t":$years_value, "int32_t":$months_value);
```

Comment:
The [spec](https://substrait.io/types/type_classes/#simple-types) only allows "a range of [-10,000..10,000] years." If we don't add a constraint now, we should add a `TODO` to be sure we add it later.